### ISSUE TITLE
SLF4J error when creating hello world template

### ISSUE THREAD

**Issue by Nekyt** [coursier/coursier#3045]
Hi, I ran sbt for the first time on windows 10 and when running I have the following error in log.

$ sbt new scala/scala3.g8

[info] resolving Giter8 0.16.2...
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

**Comment by imdaveho** [comment]
I've just encountered this as well. Possibly a quick fix or comment will help with newcomer's experience. Given OP posted this in mid 2024, I'm hoping this could get addressed quickly? Having errors or issues on official "getting started" doesn't leave a good feeling...

**Comment by tgodzik** [comment]
@adpi2 do you have any idea why this happening only on new? Is that a gitter issue?

**Comment by tgodzik** [comment]
Och, maybe we need to bump https://github.com/foundweekends/giter8/blob/7019344151f70069b3a1479b59c5e80f3a14e4fe/build.sbt#L167

Should giter8 be actually moved to sbt org?

**Comment by adpi2** [comment]
> Och, maybe we need to bump https://github.com/foundweekends/giter8/blob/7019344151f70069b3a1479b59c5e80f3a14e4fe/build.sbt#L167

Yes probably there is a mismatch between the slf4j-api version and the slf4j-simple version.

> Should giter8 be actually moved to sbt org?

[foundweekends](https://github.com/foundweekends) is probably administrated by @eed3si9n so we should be fine.

**Comment by imdaveho** [comment]
I'm new to java land and scala land, but this seems to happen only when running `sbt new`. If I run `g8 scala/scala3.g8` this warning doesn't get raised. Possibly because the logger is being specified? But I think sbt does not do this in its own dependencies, hence the warning. It's just odd that the official getting started for the most basic of things would start with some warnings of a failure to load something...even if it is not a straight up Scala or sbt thing

**Comment by eed3si9n** [comment]
I've posted my guess https://github.com/foundweekends/giter8/issues/931. Generally the problem is with the combination of slf4j, which uses classpath as a mechanism of choosing the logger, and general process of automatic dependency update. Giter8 has has multiple rounds of slf4j logger issue because when one of its dependency like JGit adds a slf4j binding the user will see "Class path contains multiple SLF4J bindings", and later when it removes the slf4j binding the user would see "SLF4J: Failed to load class". I guess a way to prevent is would be to write a test that would capture standard out. Either case, this is neither the problem of Coursier or sbt. It's Giter8 issue.
